pkg/apis/policy/v1alpha1/clustertaint_types.go

```diff
@@ -51,13 +56,59 @@ type ClusterTaintPolicySpec struct {
 	// The match conditions are ANDed.
 	// When the MatchConditions no longer match, the taints will be removed.
 	// It can not be empty.
+	// Deprecated: in favor of AddOnConditions and RemoveOnConditions.
```

Comment:
This field will be deleted after the `cluster-taint-policy-controller` is adjusted.

Reply:
Don't have to deprecate this, as no official release include this.

Reply:
Thanks, I deleted it directly.